### PR TITLE
Support more search space and maxDuration format

### PR DESCRIPTION
**Pull request by lvybriage**
support more search space;
change maxDuration and still running style.

### CODE REVIEW

File: src/webui/src/components/Sessionpro.tsx [hunk 1]
```diff
@@ -88,6 +88,20 @@ class Sessionpro extends React.Component<{}, SessionState> {
         };
     }
 
+    convertTime = (num: number) => {
+        let hour = Math.floor(num / 3600 % 24);
```
Comment (chicm-ms) [inline src/webui/src/components/Sessionpro.tsx]: why mod 24 here?  it seems there is no day unit here.

File: src/webui/src/components/Sessionpro.tsx [hunk 2]
```diff
@@ -88,6 +88,20 @@ class Sessionpro extends React.Component<{}, SessionState> {
...
+        if (num % 3600 === 0) {
+            return result = hour + 'h';
+        } 
+        let min = Math.floor(num / 60 % 60);
+        if (hour < 1) {
+            return result = min + '';
+        } else {
+            return result = hour + 'h  ' + min + 'min';
+        }
+    }
```
Comment (chicm-ms) [inline src/webui/src/components/Sessionpro.tsx]: This function can be simplified as following:
const hour =  Math.floor(num / 3600);
const min = Math.floor(num / 60 % 60);
return hour > 0 ? `${hour} h ${min} min` : `${min} min`

Comment (lvybriage) [reply src/webui/src/components/Sessionpro.tsx]: Thanks very much.

File: src/webui/src/components/Sessionpro.tsx (hunk 1 above)
Comment (lvybriage) [reply src/webui/src/components/Sessionpro.tsx]: ok

**Pull request merged**